sdcm/nemesis.py

```diff
@@ -1840,13 +1840,15 @@ def repair_nodetool_rebuild(self):
             self.target_node.run_nodetool('rebuild', retry=0)
 
     def disrupt_nodetool_cleanup(self):
-        # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
-        test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+        # Inner disrupt function for ParallelObject
+        def _dis_nodetool_cleanup(node):
             InfoEvent('NodetoolCleanupMonkey %s' % node).publish()
             with adaptive_timeout(Operations.CLEANUP, node, timeout=HOUR_IN_SEC * 48):
-                for keyspace in test_keyspaces:
-                    node.run_nodetool(sub_cmd="cleanup", args=keyspace)
+                node.run_nodetool(sub_cmd="cleanup")
+
+        threads = ParallelObject(self.cluster.nodes, num_workers=min(
+            32, len(self.cluster.nodes)), timeout=HOUR_IN_SEC * 48)
+        threads.run(_dis_nodetool_cleanup, unpack_objects=True)
```

Comment:
Why `unpack_objects=True`? Is it needed?

Reply:
Removed.